bootstrapvz/providers/azure/tasks/packages.py

```diff
@@ -47,3 +47,9 @@ def run(cls, info):
 			if os.path.isfile(info.manifest.provider['waagent']['conf']):
 				log_check_call(['cp', info.manifest.provider['waagent']['conf'],
 				                os.path.join(info.root, 'etc/waagent.conf')])
+
+		# The Azure Linux agent uses 'useradd' to add users, but SHELL
+		# is set to /bin/sh by default. Set this to /bin/bash instead.
+		from bootstrapvz.common.tools import sed_i
+		useradd_config = os.path.join(info.root, 'etc/default/useradd')
+		sed_i(useradd_config, r'^(SHELL=.*)', r'SHELL=/bin/bash')
```

Comment:
Currently the Azure Linux agent uses 'useradd' to create a new user during provisioning, but it does not specify the shell.  It leaves it up to the VM config, but the config results in a bad experience.

I suppose I could get that changed in the Linux agent, but that would require a new release.  I would argue that a default of /bin/sh isn't very good/realistic anyway, but just my opinion :)